src/uploadDsym.groovy

```diff
@@ -0,0 +1,99 @@
+import groovyx.net.http.HTTPBuilder
+import net.sf.json.JSON
+import org.apache.http.entity.mime.HttpMultipartMode
+import org.apache.http.entity.mime.MultipartEntityBuilder
+import org.apache.http.entity.mime.content.FileBody
+import org.apache.http.entity.mime.content.StringBody
+import static groovyx.net.http.Method.POST
+import com.urbancode.air.AirPluginTool;
+
+final def workDir = new File('.').canonicalFile;
+final def props = new Properties();
+final def inputPropsFile = new File(args[0]);
+try {
+    inputPropsStream = new FileInputStream(inputPropsFile);
+    props.load(inputPropsStream);
+}
+catch (IOException e) {
+    throw new RuntimeException(e);
+}
+
+final def applicationId = props['application_id']
+final def productName = props['product_name'];
+final def xcArchivePath = props['archive_path'];
+
+def dumpDwafIDs(path) {
+    def dwarfDump = "xcrun dwarfdump --uuid ${path}"
+    def tr1 = "tr '[:upper:]' '[:lower:]'"
+    def tr2 = "tr -d '-'"
+    def awk = "awk '{print \$2}'"
+    def xargs = "xargs"
+    def sed =  "sed 's/ /,/g'"
+    def cmd = "$dwarfDump | $tr1 | $tr2 | $awk | $xargs | $sed"
+
+    def p = ['/bin/bash', '-c', cmd].execute()
```

Comment:
We'll want to add Windows support just in case. It's trivial. 

If you implement AirPluginTool above as `AirPluginTool apTool = new AirPluginTool(args[0], args[1])`

Then you can use this code for line 34

`def p`
`if (apTool.isWindows) {`
    `    p = ['cmd', '/C', cmd].execute() `
`} else {`
    `    p = ['/bin/bash', '-c', cmd].execute()`
`}`